app/views/shared/page_templates/_form_page_template.html.erb

```diff
@@ -1,10 +1,14 @@
 <%= render(partial: 'shared/forms/error_summary', locals: { model: show_errors_for }) if show_errors_for %>
 <fieldset class="govuk-fieldset">
-  <legend class="govuk-fieldset__legend govuk-fieldset__legend--xl">
-    <h1 class="govuk-fieldset__heading">
-      <%= content_for(:page_title) %>
-    </h1>
-  </legend>
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <legend class="govuk-fieldset__legend govuk-fieldset__legend--xl">
+        <h1 class="govuk-fieldset__heading">
+          <%= content_for(:page_title) %>
+        </h1>
+      </legend>
+    </div>
+  </div>
```

Comment:
This corrects the change in https://github.com/ministryofjustice/laa-apply-for-legal-aid/pull/359 that broke headings on other pages.